src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/TypeNameParserHelpers.cs

```diff
@@ -438,12 +438,14 @@ internal static bool TryDive(TypeNameParseOptions options, ref int depth)
             return !IsMaxDepthExceeded(options, depth);
         }
 
-#if SYSTEM_REFLECTION_METADATA
```

Comment:
I meant all changes in this file can be reverted.

Reply:
This means I will need to add back the `Arg_NotSimpleTypeName` message.